tests/generic_config_updater/field_operation_validator_test.py

```diff
@@ -0,0 +1,142 @@
+import io
+import unittest
+import mock
+import json
+import subprocess
+import generic_config_updater
+import generic_config_updater.field_operation_validators as fov
+import generic_config_updater.gu_common as gu_common
+
+from unittest.mock import MagicMock, Mock, mock_open
+from mock import patch
+from sonic_py_common.device_info import get_hwsku, get_sonic_version_info
+
+
+class TestValidateFieldOperation(unittest.TestCase):
+
+    @patch("generic_config_updater.field_operation_validators.get_asic_name", mock.Mock(return_value="unknown"))
+    def test_rdma_config_update_validator_unknown_asic(self):
+        patch_element = {"path": "/PFC_WD/Ethernet4/restoration_time", "op": "replace", "value": "234234"}
+        assert generic_config_updater.field_operation_validators.rdma_config_update_validator(patch_element) == False
+
+    @patch("sonic_py_common.device_info.get_sonic_version_info", mock.Mock(return_value={"build_version": "SONiC.20171131"}))
```

Comment:
This should never happen in a real device. If it could use GCU feature, its  build should be 202205+

Reply:
Thanks, I changed to test BUFFER_POOL feature, which has restriction of 202211+

Reply:
Suggest to implement the limitation according to the ground truth in the above table, which has no relation to GCU feature at all.

In theory GCU feature could be backported, then we do not need to change the logic in this PR again.

Reply:
That should be the current implementation in the feature code. I just changed this test case to reflect an actual production scenario with current GCU implementation only in 202205+